### PR TITLE
Event detail view

### PR DESCRIPTION
**Pull request by Ocramius**
Presents a small (currently minimal) hierarchical view of which events where dispatched, in which order and with which nesting

Ping @Danielss89

### CODE REVIEW

File: src/Roave/DeveloperTools/Renderer/Detail/DetailEventsRenderer.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+     * @param EventInspection[] $inspections
+     *
+     * @return array
+     */
+    private function computeEventsHierarchy(array $inspections)
+    {
+        $indexed = [];
+
+        foreach ($inspections as $inspection) {
+            $indexed[$inspection->getInspectionData()[EventInspection::PARAM_EVENT_ID]] = $inspection;
```
Comment (Danielss89) [inline src/Roave/DeveloperTools/Renderer/Detail/DetailEventsRenderer.php:92]: A little bit hard to read, maybe assign `$inspection->getInspectionData()[EventInspection::PARAM_EVENT_ID]` to a var and use that var as array key

File: src/Roave/DeveloperTools/Renderer/Detail/DetailEventsRenderer.php [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    {
+        $viewModel = parent::render($inspection);
+
+        /* @var $inspections EventInspection[] */
+        $inspections = $inspection->getInspectionData();
+        $hierarchy   = $this->computeEventsHierarchy($inspections);
+        $indexed     = [];
+
+        foreach ($inspections as $inspection) {
+            $indexed[$inspection->getInspectionData()[EventInspection::PARAM_EVENT_ID]] = $inspection;
```
Comment (Danielss89) [inline src/Roave/DeveloperTools/Renderer/Detail/DetailEventsRenderer.php:74]: A little bit hard to read, maybe assign `$inspection->getInspectionData()[EventInspection::PARAM_EVENT_ID]` to a var and use that var as array key

**Comment by Danielss89** [comment]
screen shot 2014-04-21 at 23 23 10
I see a lot of data missing.
Don't know if it's missing commit or just TBD

**Comment by Ocramius** [comment]
@Danielss89 the main work is in the detail view. It is indeed still only data...

**Comment by Danielss89** [comment]
Ahh ok, i'll check that

**Comment by Danielss89** [comment]
There we go! :+1:

**Pull request merged**